crates/rooch-integration-test-runner/src/lib.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use clap::Parser;
-use clap::__macro_refs::once_cell::sync::Lazy;
+use clap::__derive_refs::once_cell::sync::Lazy;
```

Comment:
Should it be `use once_cell::sync::Lazy`?

Reply:
Deal with this later.